src/components/generic/Table/usePersistUserTablePreferences.js

```diff
@@ -4,29 +4,35 @@ import usePrevious from '../../../library/usePrevious'
 
 const usePersistUserTablePreferences = ({ key, defaultValue = undefined }) => {
   // Persist the table user preferences in sessionStorage
-  const [tableUserPreferences, setTableUserPreferences] = useSessionStorage({ key, initialValue: defaultValue })
+  const [tableUserPreferences, setTableUserPreferences] = useSessionStorage({
+    key,
+    initialValue: defaultValue,
+  })
   // Set the previous value
   const previousPreferences = usePrevious(tableUserPreferences)
 
-  const handleSetTableUserPreferences = useCallback(({ propertyKey, currentValue }) => {
-    const previousPreferencesPropExists = previousPreferences
-      && Object.prototype.hasOwnProperty.call(previousPreferences, propertyKey)
-    const valueChanged = !previousPreferencesPropExists
-      || (
+  const handleSetTableUserPreferences = useCallback(
+    ({ propertyKey, currentValue }) => {
+      const previousPreferencesPropExists =
+        previousPreferences &&
+        Object.prototype.hasOwnProperty.call(previousPreferences, propertyKey)
+      const valueChanged =
+        !previousPreferencesPropExists ||
         // Current value sometimes set to undefined by useTable after setting it.
         // Do not update preferences when this happens
-        currentValue !== undefined
-        && JSON.stringify(currentValue) !== JSON.stringify(previousPreferences[propertyKey])
-      )
+        (currentValue !== undefined &&
+          JSON.stringify(currentValue) !== JSON.stringify(previousPreferences[propertyKey]))
 
-    if (valueChanged) {
-      // Update the preference specified in the key
-      setTableUserPreferences({
-        ...tableUserPreferences,
-        [propertyKey]: currentValue || ''
-      })
-    }
-  }, [tableUserPreferences, setTableUserPreferences, previousPreferences])
+      if (valueChanged) {
```

Comment:
I didn't actually change anything in this file, vscode might have updating the spacing when I was looking at it